addon/mixins/style-bindings.js

```diff
@@ -1,6 +1,7 @@
-Ember.AddeparMixins = Ember.AddeparMixins || Ember.Namespace.create();
+// TODO(azirbel): This needs to be an external dependency.
```

Comment:
yes, it's referring to this file (style bindings) and not to Ember. I'll add a line break. We want to share `style-bindings.js` between our open source repositories